tests/sync/test_subscriber_cleanup.py

```
@@ -0,0 +1,335 @@
+    def test_newline_in_name_is_header_injection(self):
+        self.assertIn(
+            hygiene.HEADER_INJECTION, self._codes(name="Jane\nBcc: victim@example.org")
+        )
+
+    def test_mail_header_in_comments_is_header_injection(self):
+        self.assertIn(
+            hygiene.HEADER_INJECTION, self._codes(comments="Content-Type: text/html")
+        )
+
+    def test_zero_width_character_in_name_is_suspicious_unicode(self):
+        self.assertIn(hygiene.SUSPICIOUS_UNICODE, self._codes(name="Ja​ne"))
```

Ruff (0.16.0)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

Replace with escape sequence

(PLE2515)